lib/y2_installbase.pm

```diff
@@ -628,6 +628,15 @@ sub save_remote_upload_y2logs {
     $self->investigate_yast2_failure();
 }
 
+# Builds RC1, PublicRC and GMC are following the beta SDLC phase and can be identified by the following function during installation
+sub is_build_past_beta {
```

Comment:
for @manfredi's taks in https://suse.slack.com/archives/C02CLB2LB7Z/p1705568594498059
we would need something like `is_sles_in_gm_phase` in case to make the connection would help you both for naming.

Reply:
#18484 merged.